test_runner/batch_others/test_multixact.py

```diff
@@ -31,10 +28,11 @@ def test_multixact(pageserver, postgres, pg_bin, zenith_cli, base_dir):
     nclients = 3
     connections = []
     for i in range(nclients):
-        con = psycopg2.connect(pg.connstr())
+        conn = pg.connect()
         # Do not turn on autocommit. We want to hold the key-share locks.
-        con.cursor().execute('select * from t1 for key share')
-        connections.append(con)
+        conn.autocommit = False
```

Comment:
Better pass autocommit=False directly to connect. I would also added comment to connect explicitly saying "autocommit=True here by default because that's what we most commonly want"

Reply:
Done, thanks!